### PR TITLE
CellsSweeper inter-delete-batch logging

### PR DESCRIPTION
**Pull request by jeremyk-91**
**Goals (and why)**:
- Improve visibility into Sweep when deleting wide / highly versioned rows. See #2944 

**Implementation Description (bullets)**:
- Add logging into `CellsSweeper` so we log after deleting every individual batch of cells, as opposed to after the entire row for very wide rows

**Concerns (what feedback would you like?)**:
- Does this cause too much noise in the logs?
- Is this the right place?

**Where should we start reviewing?**: CellsSweeper

**Priority (whenever / two weeks / yesterday)**: two weeks; not that high, but useful for debugging

---
This change is [Reviewable](https://reviewable.io/reviews/palantir/atlasdb/2958)

### CODE REVIEW

File: atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/CellsSweeper.java [hunk 1]
```diff
@@ -82,9 +82,16 @@ public void sweepCells(
             Multimap<Cell, Long> cellTsPairsToSweep,
             Collection<Cell> sentinelsToAdd) {
         if (cellTsPairsToSweep.isEmpty()) {
+            log.info("Attempted to delete 0 cell+timestamp pairs from table {}.",
+                    LoggingArgs.tableRef(tableRef));
             return;
         }
 
+        log.info("Attempted to delete {} stale cell+timestamp pairs from table {}, and add {} sentinels.",
```
Comment (jeremyk-91) [inline atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/CellsSweeper.java]: oops, should this read `to add`?

File: atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/CellsSweeper.java [hunk 2]
```diff
@@ -82,9 +82,16 @@ public void sweepCells(
             Multimap<Cell, Long> cellTsPairsToSweep,
             Collection<Cell> sentinelsToAdd) {
         if (cellTsPairsToSweep.isEmpty()) {
```
Comment (tboam) [inline atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/CellsSweeper.java]: Quick check on correctness here: this implies if `cellTsPairsToSweep` is empty but `sentinelsToAdd` is not then we don't do anything.  Is this the correct behaviour?  Can this case ever arise?

Comment (gsheasby) [reply atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/CellsSweeper.java]: I don't think this case should ever happen.

Comment (jeremyk-91) [reply atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/CellsSweeper.java]: Confirmed:
- this is only called from `sweepBatch` in `SweepTaskRunner`
- in that method, this is only called if the batch either has size `>= deleteBatchSize` (mid-loop) which is required to be positive, or if it's non-empty (at the end).

**Pull request merged**